src/evo/deterministicmns.cpp

```diff
@@ -1120,9 +1120,8 @@ bool CDeterministicMNManager::IsProTxWithCollateral(const CTransactionRef& tx, u
         return false;
     }
 
-    const CAmount expectedCollateral = GetMnType(proTx.nType).collat_amount;
 
-    if (tx->vout[n].nValue != expectedCollateral) {
+    if (const CAmount expectedCollateral = GetMnType(proTx.nType).collat_amount; tx->vout[n].nValue != expectedCollateral) {
```

Comment:
`expectedCollateral` is not used after.

Let's change it to:
```suggestion
    if (GetMnType(proTx.nType).collat_amount != tx->vout[n].nValue) {
```